source/computer/fedora-setup.rst

```diff
@@ -171,6 +176,10 @@ Fortran 编译器，其提供了 ``gfortran`` 命令::
 Intel 软件开发工具包
 ^^^^^^^^^^^^^^^^^^^^
 
+.. note::
+
+   地震学新手可以先不安装此工具包，等日常科研中确实需要使用时再安装。
+
```

Comment:
I am thinking to move the installation into a dropdown box instead of adding a note only.

Reply:
Sounds good.

Reply:
Done in dba25ac.

Reply:
Do the same thing in the macOS guide?

Reply:
Done in dba25ac.